napari/_qt/dialogs/screenshot_dialog.py

```diff
@@ -0,0 +1,47 @@
+import os
+from pathlib import Path
+from typing import Any, Callable
+
+from qtpy.QtWidgets import QFileDialog, QMessageBox
+
+
+class ScreenshotDialog(QFileDialog):
+    def __init__(
+        self,
+        save_function: Callable[[str], Any],
+        parent=None,
+        directory=str(Path.home()),
+        qt_dialog=False,
+    ):
+        super().__init__(parent, "Save screenshot")
+        self.setAcceptMode(QFileDialog.AcceptSave)
+        self.setFileMode(QFileDialog.AnyFile)
+        self.setNameFilter("Image files (*.png *.bmp *.gif *.tif *.tiff)")
+        if qt_dialog:
+            self.setDefaultSuffix(".png")
+        self.setOption(QFileDialog.DontUseNativeDialog, qt_dialog)
```

Comment:
@Czaki do you actively plan to use the non-native dialogs? If not I don't think we need to provide this option

Reply:
i.e. we could drop the `qt_dialog` arg